docs/apache-airflow/core-concepts/operators.rst

```diff
@@ -178,7 +178,7 @@ You can pass custom options to the Jinja ``Environment`` when creating your DAG.
         },
     )
 
-See the `Jinja documentation <https://jinja.palletsprojects.com/en/2.11.x/api/#jinja2.Environment>`_ to find all available options.
+See the `Jinja documentation <https://jinja.palletsprojects.com/api/#jinja2.Environment>`_ to find all available options.
```

Comment:
Can we have intersphinx link instead, like this `:external+boto3` for jinja2?

Reply:
We already do under `+jinja2`.